requirements/tests.txt

```diff
@@ -77,41 +77,41 @@ flake8==4.0.1
 flake8-tuple==0.4.1
     # via -r requirements/tests.ini
 geocoder==1.17.5
-    # via -r requirements/base.txt
+    # via -r /Users/hannesd/gsy/gsy-framework/requirements/base.txt
 geographiclib==2.0
     # via
-    #   -r requirements/base.txt
+    #   -r /Users/hannesd/gsy/gsy-framework/requirements/base.txt
     #   geopy
 geopy==2.4.1
-    # via -r requirements/base.txt
+    # via -r /Users/hannesd/gsy/gsy-framework/requirements/base.txt
```

Comment:
I am wondering why was the path included in the generated files. Maybe you ran pip-compile with absolute and not relative paths ?

Reply:
Thanks for catching this, I also realised that on the gsy-e PR.
Let me tweak pip-compile in order to remove this (or manually remove these traces)